charts/kyverno/values.yaml

```diff
@@ -231,23 +235,6 @@ config:
   - '[ReplicaSet,*,*]'
   - '[ReportChangeRequest,*,*]'
   - '[ClusterReportChangeRequest,*,*]'
-  # exclude resources from the chart
-  - '[ClusterRole,*,{{ template "kyverno.fullname" . }}:*]'
```

Comment:
Do we need to change that ?

Reply:
They are redundant if the the default `excludeKyvernoNamespace` is true.  But since that's not guaranteed, I put these exclusions back in.